pubkey_encrypt.module

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Implements hook_form_alter().
+ */
+function pubkey_encrypt_form_alter(&$form, $form_state, $form_id) {
+  switch ($form_id) {
+    case 'user_login_form':
+      // Try to have this form pass through our custom handler first.
+      array_unshift($form['#submit'], 'custom_login_submit');
+      break;
+
+    case 'user_form':
+      // Have this form pass through our custom handler too.
+      $form['actions']['submit']['#submit'][] = 'custom_user_form_submit';
```

Comment:
"custom_user_form_submit" ??????

This should be "pubkey_encrypt_user_form_submit" or removed.